src/components/buyers/DealCard.js

```
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import "../../styles/components/buyers/DealCard.css";
+import { formatPrice } from "../../utils/common"; // Adjust the path as needed
+
+const DealCard = ({ deal }) => {
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
+
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)